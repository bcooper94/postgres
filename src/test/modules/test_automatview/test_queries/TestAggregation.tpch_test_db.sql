SET automatview.training_sample_count = 1;

select
	l_returnflag,
	l_linestatus,
	sum(l_quantity),
	sum(l_extendedprice),
	avg(l_quantity),
	avg(l_extendedprice),
	avg(l_discount),
	count(l_returnflag)
from
	lineitem
group by
	l_returnflag,
	l_linestatus;

select
	l_returnflag,
	l_linestatus,
	sum(l_quantity),
	sum(l_extendedprice),
	avg(l_quantity),
	avg(l_extendedprice),
	avg(l_discount),
	count(l_returnflag)
from
	lineitem
group by
	l_returnflag,
	l_linestatus;
