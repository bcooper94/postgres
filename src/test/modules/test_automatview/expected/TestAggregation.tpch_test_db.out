1
SELECT Matview.lineitem_l_returnflag AS Matview_lineitem_l_returnflag, Matview.lineitem_l_linestatus AS Matview_lineitem_l_linestatus, Matview.sum_lineitem_l_quantity AS Matview_sum_lineitem_l_quantity, Matview.sum_lineitem_l_extendedprice AS Matview_sum_lineitem_l_extendedprice, Matview.avg_lineitem_l_quantity AS Matview_avg_lineitem_l_quantity, Matview.avg_lineitem_l_extendedprice AS Matview_avg_lineitem_l_extendedprice, Matview.avg_lineitem_l_discount AS Matview_avg_lineitem_l_discount, Matview.count_lineitem_l_returnflag AS Matview_count_lineitem_l_returnflag FROM Matview